src/wolfsftp.c

```diff
@@ -6762,6 +6762,7 @@ int wolfSSH_SFTP_SendWritePacket(WOLFSSH* ssh, byte* handle, word32 handleSz,
                             ssh->error == WS_WANT_WRITE) {
                         return WS_FATAL_ERROR;
                     }
+                    ret = WS_FATAL_ERROR;
```

Comment:
Which value would be good for ssh->error?
Do you want to create a new value?

Reply:
None of the other calls to `SFTP_GetHeader()` are setting `ssh->error` in this failure case. We should have an error for this. Maybe WS_SFTP_BAD_HEADER.